bottle_server.py

```diff
@@ -58,6 +60,19 @@ def route_static(filepath=None):
         return template('index', root=appPath, get_url=app.get_url)
     return static_file(filepath, root=join(appPath, 'static'))
 
+@get('/version', name='version')
+def get_version():
+    version_backend=subprocess.Popen(['./shell-scripts/run-version-checker.sh', CF], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr)=version_backend.communicate()
+
+    if version_backend.returncode != 0:
+        print("Error: CheckerPrinter failed %d %s %s" % (version_backend.returncode,stdout, stderr))
+        result = json.dumps({'backend_status':'exception', 'exception_msg':'500 Server Internal Error.'})
+    else:
+        result=stdout
```

Comment:
Consistent spacing around `=`.
(Relatedly, is there an auto-formatter for Python that we could enable for this file, in a separate PR?)

Reply:
https://github.com/hhatto/autopep8#features . This auto formats to PEP8 style guide. I can add this if this is something you are looking for